Gopkg.lock

```diff
@@ -348,6 +348,61 @@
   revision = "65fb64232476ad9046e57c26cd0bff3d3a8dc6cd"
```

Comment:
can you please add UPSTREAM format to this commit?
Why do we need the second commit and why is it in this PR?